multicluster/apis/multicluster/v1alpha1/resourceimport_types.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"antrea.io/antrea/pkg/apis/crd/v1alpha2"
+	mcs "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
```

Comment:
mcs should be part of first import group

Reply:
I thought we agree that mcs would have its own directory, including resources, CRDs, etc. because this is required to use Kube-builder.

I raised the issue with @jianjuns @antoninbas, and ask for help to build skeleton if we want to mcs seamlessly merged with existing antrea framework. My understanding is that they want to go with using Kube-buidler.

But it is up to u guys to decide what is best for antrea.

Reply:
sorry I mis-read ur comments.